ElasticsearchReportHelper.php

```diff
@@ -561,6 +561,110 @@ public static function permissionFilters(array $options) {
     }
   }
 
+   /**
```

Comment:
Sorry, being ultra fussy - I think this comment will fail linting, because of an extra space before the opening / and the first line should be a one-line summary, followed by a blank line, then any more details. I'm not sure if you are able to enable phpcs for linting to Drupal standards? Main thing I noticed was to always have a space either side of = or =>.

Reply:
I'm trying to get phpcs set up, but may take a day or two because I can't install Composer until it is cleared by CSS (we have Carbon Black on Windows machines which prevents installation of unauthorised software).

Reply:
Still waiting for CCS to clear Composer for me so phpcs not set up therefore there still be linting errors. But I addressed this one.